core/database/PublisherQueryBuilder.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+class PublisherQueryBuilder
+{
+    protected $pdo;
+
+    public function __construct($pdo)
+    {
+        $this->pdo = $pdo;
+    }
+
+    public function selectAllPublishers($table)
+    {
+        //preparing the sql query
+        $statement = $this->pdo->prepare("select * from {$table}");
+        //executing the query
+        $statement->execute();
+
+        return $statement->fetchAll(PDO::FETCH_CLASS);
+    }
+
+    public function insertPublisher($table, $parameters)
+    {
+
+        $sql = sprintf(
+            'insert into %s (%s) values (%s);',
+            $table,
+            implode(', ', array_keys($parameters)), //implode turns array into a string and adds separator ', ' in between
+            ':' . implode(', :', array_keys($parameters))
+        );
+
+
+
+        try {
```

Comment:
pasiskaityk kaip veikia try/catch. Ka reiskia is vis dabar tas catch? (spoiler alert, be throw nera ir catch :) )